### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by bmukha**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import express from 'express';
+import express, { NextFunction, Request, Response } from 'express';
 import swaggerUi from 'swagger-ui-express';
-import swaggerJsDoc from 'swagger-jsdoc';
+import swaggerJsDoc, { Options } from 'swagger-jsdoc';
 
 import healthCheckRouter from './routes/health-check.js';
 import dadJokeRouter from './routes/dad-joke.js';
```
Comment (IhorOsadchyi) [inline src/index.ts]: This import looks not correct

File: package.json [hunk 2]
```diff
@@ -2,11 +2,11 @@
   "name": "epam-nodejs-practice",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "index.ts",
   "type": "module",
```
Comment (IhorOsadchyi) [inline package.json]: Not sure this configuration will work.

**Review by IhorOsadchyi — COMMENTED** [review]
What version of node was used?

File: package.json [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon src/index.js"
+    "start": "npx nodemon"
```
Comment (IhorOsadchyi) [inline package.json]: For dev server better to have something like `start:dev` script command. `start` is commonly used as default for running production build.

**Comment by bmukha** [comment]
@ihor-osadchyi-dev I use Node version 18.18.0 for this project. All the issues you have mentioned have been addressed. Please, re-review.

**Review by IhorOsadchyi — APPROVED** [review]

**Pull request merged**